PCGen-Formula/code/src/java/pcgen/base/formula/parse/Operator.java

```diff
@@ -54,7 +54,7 @@ public String getSymbol()
 
 	private String symbol;
 
-	private Operator(String op)
+	Operator(String op)
```

Comment:
I think I had some followups here, but I'm likely to spend more time on the main codebase than -formula so going to just close this